react/src/components/molecules/MainNav/index.js

```diff
@@ -75,8 +75,8 @@ class MainNav extends Component {
               });
               itemBody.push((
                 <div className={navItemClasses} key={`navItem${index}`} aria-hidden={!isExpanded}>
-                  <ul role="menu" aria-labelledby={buttonId} className="ma__main-nav__container" tabIndex={!isExpanded ? -1 : null}>
-                    <li role="presentation" className="ma__main-nav__subitem">
+                  <ul role="menu" aria-label={`Submenu of ${buttonId}`} className="ma__main-nav__container">
+                    <li role="menuitem" className="ma__main-nav__subitem">
```

Comment:
doesn't yukiko want this to be presentation role based on the ticket information below?

Change aria role on li.ma__main-nav__subitem.
Before
```
<ul role="menu" aria-labelledby="button0" class="ma__main-nav__container" tabindex="-1">
...
  <li role="menuitem" class="ma__main-nav__subitem">
...
```

After
```
<ul role="menu" aria-labelledby="button0" class="ma__main-nav__container" tabindex="-1">
...
  <li role="presentation" class="ma__main-nav__subitem">
...
```

Reply:
Good catch, fixing!